spec.html

```diff
@@ -7751,8 +7751,8 @@ <h1>HostCallJobCallback ( _jobCallback_, _V_ [ , _argumentsList_ ] )</h1>
       <p>HostCallJobCallback is a host-defined abstract operation that argument _jobCallback_ (a JobCallback Record), _V_ (an ECMAScript language value) and optional argument _argumentsList_ (a List of ECMAScript language values).</p>
       <p>The implementation of HostCallJobCallback must conform to the following requirements:</p>
       <ul>
-        <li>If _argumentsList_ is not present, it must always return the result of Call(_jobCallback_.[[Callback]], _V_).</li>
-        <li>If _argumentsList_ is present, it must always return the result of  Call(_jobCallback_.[[Callback]], _V_, _argumentsList_).</li>
+        <li>If _argumentsList_ is not present, it must always perform and return the result of Call(_jobCallback_.[[Callback]], _V_).</li>
```

Comment:
tfw not allowed to use ai to predict the result of Call() instead of performing it

Reply:
if it's unobservable that you've skipped it then you certainly could avoid performing it :-p